tests/test_data.py

```diff
@@ -38,6 +38,12 @@
 userPermissionIdentifier = "user"
 permissionNames = ["VIEW", "EDIT", "DELETE"]
 permissionGroupId = '24231-52321-43523-64353-23111'
+username = "testuser"
+permission_ids = [1,2,3]
+user_type = "user"
+group_type = "group"
```

Comment:
I think either this or userPermissionIdentifier and groupPermissionIdentifier can be used

Reply:
Done